### ISSUE TITLE
as pycrypto is abandonded,would you like to move to pycryptodome?

### ISSUE THREAD

**Issue by Anda001** [NuCOS/nucosCR#1]
rt. 
Use cryptography instead, pycrypto is abandoned and not maintained.
and the pycrypto  is hard to build throuth.

https://github.com/pycrypto/pycrypto/issues/303
https://github.com/pycrypto/pycrypto/issues/221

**Comment by DocBO** [comment]
Yes, Thanks for the comment. I think this would be the best way.
